selenium/manifest.yaml

```diff
@@ -0,0 +1,8 @@
+---
+name: Selenium
+version: 3.141.59
+maintainer: "@scndel"
+description: Selenium is a web browser testing framework. This app deploys a ready hub with Chrome node.
+PLEASE PROVIDE AT LEAST 2 GB MEMORY FOR STABILITY.
```

Comment:
This needs to be on one line - it isn't valid YAML this way - that's why the tests failed. Thanks.

Reply:
Would it be better if it wasn't in all caps?